examples/data/simpack_xrd_AlGaAs.xrdml

```diff
@@ -0,0 +1,121 @@
+			</dataPoints>
+		</scan>
+	</xrdMeasurement>
+</xrdMeasurements>

```

Comment:
This file could be somehow compressed to save space. Most compressions can be read transparently by xrayutilities. see for example here: https://xrayutilities.sourceforge.io/examples.html#reading-xrdml-files